configuration/initialization.sh

```diff
@@ -23,9 +23,12 @@ latest_version=$(echo "$release_info" | grep -o '"tag_name": *"[^"]*"' | cut -d
 # Fetching all tags from Git repository
 tags=$(git ls-remote --tags https://github.com/usdot-fhwa-OPS/V2X-Hub.git | awk -F/ '{print $3}' | sort -V)
```

Comment:
On issue I am realizing right now is that getting tags may return tags that are not associated with any release. Maybe we should considered install `gh' as part of this script and using the output of  `gh release list` which looks like this 
![image](https://github.com/usdot-fhwa-OPS/V2X-Hub/assets/77466294/6f37b6a3-547f-4ed6-a990-af4668c58db5)

Reply:
I think using "gh" would require an additional step of installing it. As part of this bug, we can work on removing the unwanted characters and add a follow up story to make an enhancement.

In the follow up story, we can add a logic to check if gh installed or not and add installation commands within this script.